package.json

```diff
@@ -3,6 +3,12 @@
   "version": "1.4.2",
   "description": "Open color scheme for web UI",
   "main": "open-color.scss",
+  "files": [
```

Comment:
key main target to root open-color.scss, files is for other files, dont need include the main again